plugins/tech-radar/src/components/RadarPage.tsx

```diff
@@ -45,11 +45,27 @@ export const RadarPage = ({
   ...props
 }: TechRadarPageProps): JSX.Element => {
   const classes = useStyles();
+  const [searchText, setSearchText] = React.useState('');
+
+  const searchInput = (
```

Comment:
should probably `useCallback` this one, and give the result directly to the `onChange` instead of via a `e => {}` so that the text field always gets the same callback value on each paint